### PR TITLE
chore: add vs code workspace

### PR DESCRIPTION
**Pull request by nicojs**
The current developer workflow is a bit all-over-the-place. With this PR, I want to make an end to this.

Meet: VS Code workspaces

## Goal

Create a single developer experience for all core contributes.

Open the workspace with `$ code workspace.code-workspace`

## Features

All code in one single vscode instance!
image

Yet... all build tasks right there when pressing CTRL+B (or F1 "Run build task")
image

And... all run tasks right there in the drop down! 
image

### CODE REVIEW

**Comment by nicojs** [comment]
@wmeints do you also want to take a look at this PR? Is this enough documentation to get devs started with the vscode workspace?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,22 +4,42 @@ This is the contribution guide for Stryker. Great to have you here! Here are a f
...
+
+We've chosen to **check in in our vscode configuration**. This makes development unified amongst stryker developers. VSCode is an open source code editor maintained by Microsoft. For more info and the download link, please visit https://code.visualstudio.com/.
+
+After cloning this repo, open the workspace with `$ code workspace.code-workspace` (or open code and use file -> Open Workspace...).
+
+Some quick notes to help you get started:
+
+1. On the left side, you can see all stryker projects and plugins. Open files from there.
+1. Use `CTRL+B` (or `⌘+B` on OSX) to open up the *Run build task* menu. This can enable typescript compilation + watch for a plugin directory.
+1. Use `CTRL+Shift+D` (or `⌘⇧D` on OSX) to open up the *debug* pane. Here you can select a config to run. For example: select "Unit tests (stryker-api)" to run the unit tests for the `stryker-api` package.
```
Comment (Archcry) [inline CONTRIBUTING.md]: I currently don't see the option to run unit tests for stryker-api when trying to follow this step. Is it just me?

Comment (Archcry) [reply CONTRIBUTING.md]: Nvm, I found the solution, `code .` does not work, you have to explicitly use `code workspace.code-workspace` unfortunately.

**Comment by Archcry** [comment]
Good work!

**Pull request merged**